__test__/pet.test.js

```diff
@@ -0,0 +1,155 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+});
+
+describe('constructor', () => {
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
+    });
+});
+
+describe('constructor', () => {
+    it('has an initial age of 0', () => {
+        const pet = new Pet('Fido');
+        expect(pet.age).toEqual(0);
+    });
+});
+
+describe('constructor', () => {
+
+    it('throws an error if the pet is not alive', () => {
+        const pet = new Pet('Fido');
+
+        pet.age = 30;
+
+        expect(() => pet.growUp()).toThrow('Your pet is no longer alive :(');
+    });
+
+    it('increments the age by 1', () => {
+        const pet = new Pet('Fido');
+        pet.growUp();
+        expect(pet.age).toEqual(1);
+    });
+});
+
+describe('constructor', () => {
+    it('increases the hunger property by 5', () => {
+        const pet = new Pet('Fido');
+        pet.growUp();
+        expect(pet.hunger).toEqual(5);
+    });
+});
+
+describe('constructor', () => {
+    it('decreases fitness by 3', () => {
+        const pet = new Pet('Fido');
+        pet.fitness = 10;
+        pet.growUp();
+        expect(pet.fitness).toEqual(7);
+    });
+});
+
+describe('walk', () => {
+
+    it('throws an error is the pet is not alive', () => {
+        const pet = new Pet('Fido');
+
+        pet.fitness = 0;
+
+        expect(() => pet.walk()).toThrow('Your pet is no longer alive :(');
+    });
+
+    it('increases fitness by 4 to a maximum of 10', () => {
+        const pet = new Pet('Fido');
+
+        pet.fitness = 8;
+        pet.walk();
+
+        expect(pet.fitness).toEqual(10);
+    });
+});
+
+describe('feed', () => {
+
+    it('throws an error is the pet is not alive', () => {
+        const pet = new Pet('Fido');
+
+        pet.age = 30;
+
+        expect(() => pet.feed()).toThrow('Your pet is no longer alive :(');
+    });
+
+    it('decreases hunger level by 3 to a minimum of 0', () => {
+        const pet = new Pet('Fido');
+
+        pet.hunger = 5;
+        pet.feed();
+        expect(pet.hunger).toEqual(2);
+    });
```

Comment:
Similar to `walk`, this test only tests that hunger decreases by 3, not that it decreased to a minimum of 0. Using the example I gave above for `walk` could you separate this out into 2 tests, one for testing that hunger decreased by 3 and one that tests that hunger decreases to a minimum of 0.